input/postgres/replication.go

```diff
@@ -76,12 +76,49 @@ SELECT client_addr,
 	FROM %s
  WHERE client_addr IS NOT NULL`
 
+const transactionIdSQLPg13 string = `
+SELECT
+	pg_current_xact_id(),
```

Comment:
[note for reviewer]
`pg_current_xact_id()` will assign a new transaction ID if the current transaction does not currently have one. This means, it'll likely consume/assign a new transaction ID. There would be an argument of consuming such thing, but full snapshot only runs every 10 mins and it will likely be a small increase of transaction ID (144 per day) which will very less likely to be any issues to the Postgres server, so we choose to use this function.